install/kubernetes/helm/istio-init/values.yaml

```diff
@@ -5,7 +5,7 @@ global:
   hub: gcr.io/istio-release
 
   # Default tag for Istio images.
-  tag: master-latest-daily
+  tag: release-1.2-latest-daily
```

Comment:
have to take care with this tag - there is related code in the infrastructure that does a sed replacement on it.  Has an accompanying PR been submitted to test-infra?

Reply:
I'd like @rlenglet to chime in on this one.

Reply:
@sdake Branch `release-1.1` uses tag `release-1.1-latest-daily`. Why would `release-1.2` be different?

Reply:
@sdake if you were referring to `release/gcb/helm_values.sh`, it should already work:
```
sed -i "s|tag: .*-latest-daily|tag: ${CB_VERSION}|g"         ./"istio-${CB_VERSION}"/install/kubernetes/helm/istio*/values.yaml
```